modules/op-parameters-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -1,11 +1,16 @@
 // This module is included in the following assemblies:
-// * pac/managing-pipeline-runs-pac.adoc
+// * pac/creating-pipeline-runs-pac.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="parameters-pipeline-run-using-pipelines-as-code_{context}"]
-= Parameters in a pipeline run specification
+= Dynamic variables in a pipeline run specification
 
-You can use parameters in a pipeline run specification to provide information about the commit that triggered the pipeline run and to use the temporary GitHub App token for Github API operations.
+You can use dynamic variables in a pipeline run specification to provide information about the commit that triggered the pipeline run and to use the temporary GitHub App token for Github API operations.
+
+[IMPORTANT]
+====
+Using {pac} dynamic variables in default values for pipeline or task parameters is currently not supported. You can use a dynamic variable in a `value:` field, but not in a `default:` field.
```

Comment:
this has been fixed in https://issues.redhat.com/browse/SRVKP-4070 merged in https://github.com/openshift-pipelines/pipelines-as-code/commit/7c3294d1e08b07b5d7cc8a3b086cf00dd49041e3 so you can remove that

Reply:
but that would be released in 1.19.x i think, so i don't know which version these docs target to....

Reply:
This PR targets 1.18. I have a separate doc task for the fix. As I understand the text is fine for 1.18, so leaving unchanged